run_python_examples.sh

```diff
@@ -154,6 +154,9 @@ function vision_transformer() {
 
 function word_language_model() {
   uv run main.py --epochs 1 --dry-run $CUDA_FLAG --mps || error "word_language_model failed"
+  for model in "RNN_TANH" "RNN_RELU" "LSTM" "GRU" "Transformer"; do
```

Comment:
This change is unrelated with the PR. Remove. Or just rebase on top of latest main.